DEADMAN_155/Deadman_Check.py

```diff
@@ -0,0 +1,370 @@
+#! /usr/local/bin/python
+#
+# Deadman.py - Deadman Timing Checker
+#
+"""
+
+Guideline: 
+https://occweb.cfa.harvard.edu/twiki/bin/view/Constraints/MPGuidelines/Development/GuidelineRadiationSafingWithoutHRC
+
+
+      Ensure the following is commanded for every rad entry Priority 2: 
+
+      Time 	                   Event 	                       Source
+============================================================================================
+Rad Entry Minus 48h 	Activate SCS-155 containing; 	    Weekly Loads (via fot request)
+                         A_NSM_XXHR.RTS timer = 48h, 10m
+                         
+Rad Entry 	              Nominal SI Safing              Weekly Loads (via ORViewer)
+Rad Entry + 5m 	              Disable SCS-155                Weekly Loads (via fot request)
+
+IMPORTANT: VO (Vehicle Only) loads DO NOT HAVE OORMPDS or OORMPEN
+              VO loads DO have EQF013M and EEF1000
+           VpS (Vehicle plus Science) loads have EQF013M and EEF1000
+           So this program focuses on using EQF013M and EEF1000 to determine Radzone
+           entry for all loads.
+
+  There are 4 values you need in order to assess the timing of the SCS-155 Deadman commands:
+
+    SCS-155 enable
+    SCS-155 activate
+    Radzone Entry
+    SCS-155 deactivate.
+
+    In this progam these 4 commands or states will be referred to as Deadman 155 states.
+    The complete set of 4 states will be referred to as a state group.
+
+  The Deadman timeout is fixed at 48 hours 10 minutes from activation.
+
+"""
+import argparse
+import numpy as np
+import sys
+
+# Import the BackstopHistory class
+from backstop_history import BackstopHistory
+
+# ACIS Ops Imports
+import apt_date_secs as apt
+import Calc_Delta as cd
+import ORP_File_Class as ofc
+
+#
+# Parse the input arguments
+#
+deadman_parser = argparse.ArgumentParser()
+deadman_parser.add_argument("review_load_path", help="Path to the Review load directory. e.g. /data/acis/LoadReviews/2022/FEB2122/ofls'")
+
+deadman_parser.add_argument("nlet_file_path", help="Path to the NLET file to be used in assembling the history. e.g. /data/acis/LoadReviews/TEST_NLET_FILES/FEB2122A_NonLoadTrackedEvents.txt")
+
+args = deadman_parser.parse_args()
+
+#
+# Inits
+# 
+# Create an instance of the Backstop History class
+BSC = BackstopHistory.Backstop_History_Class('ACIS-Continuity.txt',
+                                             args.nlet_file_path,
+                                             args.review_load_path,
+                                             0)
+# Create an instance of the ORP File Class
+ofci =  ofc.ORP_File_Class()
+
+load_week_path = args.review_load_path
+nlet_file_path = args.nlet_file_path
+
+# Extract the load week out from the path
+load_week = load_week_path.split('/')[5]
```

Comment:
Same as the question I asked in `HETG_Check.py`--will this index always be `5`?

Reply:
Yes it will always be 5 in LR world.